static/src/js/util/collections.js

```diff
@@ -220,6 +221,10 @@ export const buildCollectionSearchParams = (params) => {
   // Only include has_granules_or_cwic sort key if the parameter is being used
   if (hasGranulesOrCwic) sortKey.unshift('has_granules_or_cwic')
 
+  if (selectedSortKey[0] !== collectionSortKeys.recentVersion) {
```

Comment:
This custom logic does not seem right we don't do this for other sort keys

Reply:
This is because we are appending the sort key of `recentVersion` to the sort key array in all cases, unless we have already selected the `recentVersion` sort key as our primary method of sorting. We only do this for this key because we only want to add `recentVersion` as the secondary sort method for other sort keys.